### PR TITLE
Add partially meeting controller spec tests

### PR DESCRIPTION
**Pull request by rhiroyuki**
This PR partially solves #357.

Covered methods:
```
MeetingsController:
GET #show
GET #delete_comment
GET #new
POST #comment
GET #edit
```

### CODE REVIEW

File: app/controllers/meetings_controller.rb [hunk 1]
```diff
@@ -57,7 +57,7 @@ def delete_comment
       Notification.where(uniqueid: public_uniqueid).destroy_all
     end
 
-    render nothing: true
+    head :ok
```
Comment (julianguyen) [inline app/controllers/meetings_controller.rb]: Could you explain why you changed this and if this behaviour is equivalent to what was previously there?

File: spec/controllers/meetings_controller_spec.rb [hunk 2]
```diff
@@ -12,5 +12,240 @@
...
+          before do
+            meeting_member.destroy
+            get :show, params: { id: meeting.id }
+          end
+
+          it { expect(response).to have_http_status(:redirect) }
+          it { expect(response).to redirect_to(group_path(meeting.groupid)) }
+        end
+
+        context 'inexistent meeting id' do
```
Comment (julianguyen) [inline spec/controllers/meetings_controller_spec.rb]: replace inexistent with non-existent :P

File: app/controllers/meetings_controller.rb (hunk 1 above)
Comment (rhiroyuki) [reply app/controllers/meetings_controller.rb]: When I wrote the tests for delete_comment I got this warning.
```
 .DEPRECATION WARNING: `:nothing` option is deprecated and will be removed in Rails 5.1. Use `head` method to respond with empty response body. (called from delete_comment at ...ifme/app/controllers/meetings_controller.rb:60)
```

Searching for solutions I found this answer explaining to use 'head :ok'. 
https://stackoverflow.com/questions/34688726/the-nothing-option-is-deprecated-and-will-be-removed-in-rails-5-1

Comment (julianguyen) [reply app/controllers/meetings_controller.rb]: Oooh awesome catch/find!

**Pull request merged**